### ISSUE TITLE
Using Implicit Solvers

### ISSUE THREAD

**Issue by jpfairbanks** [AlgebraicJulia/Decapodes.jl#203]
@lukem12345, @GeorgeR227, when Chris R. got the autodiff working for implicit solvers, we tested some of our stiffer problems with those solvers. But when I checked the tests and examples just now. I only saw `Tsit5()`. Can we use other solvers?

**Comment by lukem12345** [comment]
Yes. For FEED I have gotten good results from the FBDF and Trapezoid solvers.

**Comment by jpfairbanks** [comment]
Ok, so let's make sure to update the docs so that they are using one of the implicit solvers https://docs.sciml.ai/DiffEqDocs/stable/solvers/ode_solve/#Stiff-Problems

**Comment by lukem12345** [comment]
The Calibration docs page uses the semi-implicit `Rodas5p` solver:
https://algebraicjulia.github.io/Decapodes.jl/dev/calibrate/calibration/#Sparsity-Detection-and-Jacobian-Coloring
